code/modules/mob/living/simple_animal/hostile/giant_spider/effects.dm

```diff
@@ -44,6 +45,23 @@
 	health -= damage
 	healthcheck()
 
+/obj/effect/spider/attack_hand(var/mob/living/carbon/human/user)
+	if(user.a_intent == I_HURT)
+		user.delayNextAttack(8)
+		user.visible_message("<span class='danger'>[user.name] claws at \the [src]!</span>", \
+			"<span class='danger'>You claw at \the [src]!</span>", \
+			"You hear something tear.")
+		health -= 2
+		healthcheck()
+	else
+		var/atom/movable/I = pick(contents)
+		var/some_suffix = "thing"
+		if(I && istype(I,/mob/living/carbon/human))
+			some_suffix = "one"
+		user.visible_message("<span class='notice'>[user] rubs their hands all over \the [src]!</span>", \
+			"<span class='notice'>You rub your hands over \the [src] [I ? ", you think you can feel some[some_suffix] in there!": ""]</span>")
```

Comment:
I think you could do `[I &&  ", you think you can feel some[some_suffix] in there!"]`.